sugoi-api-rest-services/src/main/java/fr/insee/sugoi/services/controller/UserController.java

```diff
@@ -466,7 +439,8 @@
       @Parameter(description = "User's id to delete", required = true) @PathVariable("id")
           String id) {
 
-    return deleteUsers(realm, null, id);
+    User foundUser = userService.findById(realm, null, id);
```

Comment:
NPE when user is not found. Fix should fix get404WhenNoUserIsFoundWhenGetById test

Reply:
yep there is no null pointer but there is still an error ```Should respond 404 ==> Unexpected exception type thrown ==> expected: <fr.insee.sugoi.core.exceptions.EntityNotFoundException> but was: <org.springframework.web.util.NestedServletException>```. Why ? because the advice controller are not in this module so the exception is not properly by the advice controller.

Reply:
Adding classes = {UserController.class, SugoiAdviceController.class} to @SpringBootTest should make it found the SugoiAdviceController class. But I really think this NPE should be fix as well.